.kitchen.yml

```diff
@@ -1,32 +1,9 @@
 ---
-driver:
-  name: vagrant
-
 provisioner:
   name: chef_solo
-
-platforms:
-  - name: ubuntu-12.04
-    attributes:
-      osl-php:
-       packages:
-           - apc:
-               php-apc: false
-           - allthepackages:
-             - php5-fpm
-             - php5-gd
-           - vim
-    run_list:
-      - recipe[apt]
-  - name: centos-6.6
-    attributes:
-      osl-php:
-       packages:
-         apc:
-             php-apc: false
-         moar_packages:
-           - php-fpm
-           - php-gd
```

Comment:
@fahlmant these attributes need to still be included so that the tests work properly.